src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,158 @@
+                break;
+            case "myTaskToday":
+                myDuke.ui.echoTaskToday(myDuke.tasklist.listUsed);
+                break;
+            case "myTaskOn":
+                myDuke.ui.echoTaskThisDay(myDuke.tasklist.listUsed,
+                        LocalDateTime.parse(command.commandContent, DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm")));
+                break;
+            case "find":
+                myDuke.ui.echoPrintFindResult(myDuke.tasklist.listUsed, command.commandContent);
+                break;
+            default:
+                try {
+                    executeFalseCommand(command.commandContent);
+                } catch (DukeException err) {
+                    myDuke.ui.echoErrMsg(err);
+                }
+            }
+        }
+    }
+
+    /**
+     * Perform 3 actions: add new task to taskList, show results with user and change taskFile content correspondingly.
+     *
+     * @param duke            The duke instance related to this action
+     * @param taskDescription The desciption of the task that we want to add
+     */
+    public static void addToList(Duke duke, String taskDescription) {
+        Task task = duke.tasklist.addTask(taskDescription);
+        duke.storage.updateFile(duke.tasklist.listUsed);
+        duke.ui.echoAddToList(task, duke.tasklist.listUsed.size());
+    }
+
+    /**
+     * Perform 3 actions: mark a task in taskList as Done, show results with user and
+     * change taskFile content correspondingly.
+     *
+     * @param duke  The duke instance related to this action
+     * @param index The index of that task in taskList
+     */
+    public static void markTaskDone(Duke duke, int index) {
+        Task task = duke.tasklist.doneTask(index);
+        duke.storage.updateFile(duke.tasklist.listUsed);
+        duke.ui.echoMarkTaskDone(task);
+    }
+
+    /**
+     * Perform 3 actions: delete a task in taskList, show results with user and
+     * change taskFile content correspondingly.
+     *
+     * @param duke  The duke instance related to this action
+     * @param index The index of that task in taskList
+     */
+    public static void deleteTask(Duke duke, int index) {
+        Task task = duke.tasklist.listUsed.get(index - 1);
+        duke.ui.echoDeleteTask(task);
+        duke.tasklist.deleteTask(index);
+        duke.storage.updateFile(duke.tasklist.listUsed);
+    }
+
+    /**
+     * Tell Duke that this command format is wrong,
+     * let it execute the action of throwing an exception to catch and handle.
+     *
+     * @param command The short syntax of the wrong format command
+     * @throws DukeException Special exception related to wrong format of command entered into Danh's Duke
+     */
+    public static void executeFalseCommand(String command) throws DukeException {
+        if (command.startsWith("list")) {
+            throw new DukeException("     list command should not have body, Sir!");
```

Comment:
Can I ask if the spacings in front of your string output was intended? This seems to be done on purpose and I have seen it quite often. Is it for UI purposes?